ci/wait-cluster.sh

```diff
@@ -4,5 +4,5 @@ echo "Waiting for Docker..."
 until docker version; do sleep 1; done
 
 echo "Waiting for cluster autovault..."
-docker exec cluster ./cluster.py wait
+docker exec cluster /bin/bash -c "echo -e '[nomad_meta]\nvolumes=/opt/volumes\n' >> cluster.ini; cat cluster.ini ;./cluster.py wait"
```

Comment:
This has no effect, since you'd need to restart the container. Let's add this config to the defaults, so remove this line.